cmd/empire/factories.go

```diff
@@ -177,6 +184,27 @@ func newKinesisLogsStreamer(c *cli.Context) (empire.LogsStreamer, error) {
 	return empire.NewKinesisLogsStreamer(), nil
 }
 
+// Events ==============================
+
+func newEventStream(c *cli.Context) (empire.EventStream, error) {
```

Comment:
I think that would be worth it if we pull the factories out into a separate package, but that gets cumbersome because we have to define Config structs for everything. Some factories require [more than 1 config](https://github.com/remind101/empire/blob/master/cmd/empire/factories.go#L83-L118) value, so this keeps things easy to change without needing to updating config structs/method definitions.